src/instantsearch/instantsearch.ts

```diff
@@ -14,13 +14,15 @@ import { isPlatformBrowser } from '@angular/common';
 
 import * as algoliasearchProxy from 'algoliasearch/lite';
 import instantsearch from 'instantsearch.js/es';
-import { AlgoliaSearchHelper } from 'algoliasearch-helper';
 
 import { Widget } from '../base-widget';
 import { VERSION } from '../version';
+import { InstantSearchOptions, InstantSearch } from 'instantsearch.js/es/types';
 
+// TODO: do we need this?
 const algoliasearch = algoliasearchProxy.default || algoliasearchProxy;
```

Comment:
I don't know what's needed here

Reply:
IIRC that's because build systems interpret `import * as algoliasearchProxy from 'algoliasearch/lite'` differently. This proxy was needed to make sure that we import the right entry point of the package.

We probably won't need that with the Search Client v4.

Reply:
cc https://github.com/algolia/angular-instantsearch/pull/707

Reply:
I mean that the `algoliasearch` variable isn't used